ISCSI/SCSI/SCSICommandDescriptorBlock/MMC/GetConfigurationCommand.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Utilities;
+
+namespace SCSI
+{
+    internal class GetConfigurationCommand : SCSICommandDescriptorBlock

```

Comment:
Shouldn't this inherit from SCSICommandDescriptorBlock10? and than you can use some of the fields...